### DISCUSSION TITLE
How to "deactivate" the database (Prisma) component?

### DISCUSSION THREAD

**Discussion by richard-hanxu** [chen-rn/CUA#29] [Q&A]
Hi,

If I want to create a page that doesn't require auth/databases, how do I configure the project so that it no longer tries to connect to a database? I thought it might not be a noticeable problem if I just ignored it, as the only effect was this message loading whenever I interact with the site after calling "yarn web".

<img width="593" alt="image" src="https://user-images.githubusercontent.com/49998382/218008027-61f949b3-54cd-44ac-9d6b-bf0e58f7cf35.png">

However, occasionally the screen goes blank and is replaced by the following,

<img width="960" alt="image" src="https://user-images.githubusercontent.com/49998382/218008551-08baf644-58a7-4e79-a42f-465550ccb93a.png">

How do I remove the database portion so that this message doesn't appear anymore?

**Comment by chen-rn** [comment]
Just so I completely understand, the problem is that for pages that don't require any interaction with a DB/auth(ie. landing page), when your DB is not live, you should theoretically get no errors, but you're getting a blank screen error right?

It seems that the problem is that somewhere in your app, you're doing a trpc call to the backend, which contains a prisma.entry.findMany() function. So, to solve this, it sounds obvious, but I'd recommend finding where you're making that call, and ensuring that your app isn't calling the db when you aren't connected! If you have not modified much of the template, it could be the following line in HomeScreen()
` const { data, isLoading, error } = trpc.entry.all.useQuery();`

Since this isn't a real-time database, your client is not directly establishing a _connection_ with the database. We're just making HTTP calls to our backend, which is "connected" to the DB. So, if you just remove all of the HTTP request attempts in the frontend this should be resolved!

note: sry this response took a while, didn't realize there was a post in Discussions 🤐
